src/main/java/randoop/reflection/TypeInstantiator.java

```diff
@@ -98,6 +99,140 @@ public TypedClassOperation instantiate(TypedClassOperation operation) {
     return operation;
   }
 
+  /**
+   * Creates a set of instantiations of a given operation that is generic or has wildcard types.
```

Comment:
Is this an ordered set?  If not, does that make Randoop nondeterministic?

Reply:
Doesn't appear to be ordered.  Perhaps 'result' should be a LinkedHashSet?